addons/hr_holidays/tests/test_holidays_calendar.py

```diff
@@ -42,3 +42,19 @@ def test_hours_time_off_request_calendar_view(self):
         self.assertEqual(last_leave.date_from.weekday(), 3, "It should be Thursday")
         self.assertEqual(last_leave.date_from.hour, expected_leave_start, "Wrong start of the day")
         self.assertEqual(last_leave.date_to.hour, expected_leave_end, "Wrong end of the day")
+
+    def test_time_type_other(self):
+        leave_type = self.env['hr.leave.type'].create({
+            'name': 'Home office',
+            'time_type': 'other',
+            'requires_allocation': False,
+        })
+        self.leave_1 = self.env['hr.leave'].create({
+            'name': 'Home working',
+            'employee_id': self.employee_hruser_id,
+            'holiday_status_id': leave_type.id,
+            'date_from': date(2020, 4, 2),
+            'date_to': date(2020, 4, 2),
+        })
+        self.leave_1.action_approve()
+        self.assertFalse(self.env['account.analytic.line'].search([("holiday_id", "=", self.leave_1.id)]))
```

Comment:
test should be moved to same module as the fix, for example here `hr_holidays` doesn't depend on `account.analytic.line` so if you run Single App test for `hr_holidays` this line will crash

Reply:
to run Single App Test locally you can do:
`./odoo-bin --addons-path=... -d [new-db] --test-enable --stop-after-init --log-level=test --max-cron-threads=0 -i [module_name] --test-tags=/[module_name]`
`module_name=hr_holidays` in this case